src/app/screens/App/shared/components/LoggedOutHeader.js

```diff
@@ -1,37 +1,34 @@
 import React from 'react';
-import { Navbar, Nav, NavItem, Button, FormGroup, FormControl, InputGroup } from 'react-bootstrap';
+import { Navbar, Nav, NavItem, Button, FormGroup, FormControl } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
 
 const LoggedOutHeader = () => (
     <Navbar>
         <Nav>
-            <NavItem disabled>Personal</NavItem>
-            <NavItem disabled>Open source</NavItem>
-            <NavItem disabled>Business</NavItem>
-            <NavItem disabled>Explore</NavItem>
-            <NavItem disabled>Help</NavItem>
+            <NavItem>Personal</NavItem>
+            <NavItem>Open Source</NavItem>
+            <NavItem>Business</NavItem>
+            <NavItem>Explore</NavItem>
+            <NavItem>Personal</NavItem>
```

Comment:
It does change their style though.  It really doesn't matter, I shouldn't have actual disable links there, I just thought it would be a visual indication that they need to be fixed.